arrays/2.provide-a-function-which-outputs-the-first-element-of-a-passed-array.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+function returnFirstElement(array, endOfArray) {
+    var values;
+    if(endOfArray !== undefined){
```

Comment:
This will work - but better solution is to set endOfArray to 1 if it is null/undefined with an || statement. That removes this if and makes the code a two-line function.